sources/console.c

```diff
@@ -28,6 +28,7 @@ enum
 	OD_LFRONTEND,
 	OD_LROUTER,
 	OD_LVERSION,
+	OD_LHARAKIRI,
```

Comment:
Do we really really want it?
Should it produce core dump or just exit?

Anyway, I think we should separate this functionality from curing memory leak which is many times more important.